app/javascript/react/components/ParksShowContainer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const ParksShowContainer = () => {
+  return(
+    <div>
+    </div>
+  )
+}
+
+export default ParksShowContainer
```

Comment:
NO. 

Not a complete feature, remove this component from the app until you are actually working on the show page